.ci/env/openblas.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#===============================================================================
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+sudo apt-get update
+sudo apt-get install libopenblas-dev
+export OPENBLASROOT=/usr/lib/x86_64-linux-gnu/openblas-pthread
```

Comment:
Will not work. Expected libopenblas.xxx in $(OPENBLASDIR)/lib.Could it be fixed by a soft link?

Reply:
If not, should be handled in makefile (TBD)

Reply:
Includes are also expected to be located in $OPENBLASROOT/include